### PR TITLE
TUNE 310: Service directory

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@saeedmb8 please review 

This is links to access functionality has been removed from the UI but the functionality is not disabled

screenshot 2017-02-07 12 38 51

File: tuneme/templates/new/servicedirectory/organisation_detail.html [hunk 1]
```diff
@@ -72,11 +72,6 @@ <h3 class="service-directory__title">Trading Hours</h3>
 
     <a class="call-to-action__nav-item call-to-action__nav-item--report" href="{% url 'molo.servicedirectory:organisation-report' organisation.id %}?org_name={{ organisation.name | urlencode }}">Is this info correct?</a>
 
-    <a class="call-to-action__nav-item" href="{% url 'molo.servicedirectory:organisation-sms' organisation.id %}" class="service-dir__button">Send Clinic Info to Someone</a>
-
-    <a class="call-to-action__nav-item" href="{% url 'molo.servicedirectory:organisation-sms-self' organisation.id %}" class="service-dir__button service-dir__button--outline">Send Clinic Info to Yourself</a>
-    <hr/>
```
Comment (smarzban) [inline tuneme/templates/new/servicedirectory/organisation_detail.html]: @Mitso shouldn't we keep the hr tag?

**Comment by smarzban** [comment]
@Mitso small comment otherwise 👍

File: tuneme/templates/new/servicedirectory/organisation_detail.html (hunk 1 above)
Comment (Mitso) [reply tuneme/templates/new/servicedirectory/organisation_detail.html]: Nice catch, thanks @saeedmb8

**Pull request merged**